pandas/io/parsers.py

```diff
@@ -1355,6 +1392,23 @@ def _convert_types(self, values, na_values, try_num_bool=True):
 
         return result, na_count
 
+    def _cast_types(self, values, cast_type, column):
+        """ cast column to type specified in dtypes= param """
```

Comment:
While I know this isn't done too much for internal functions, it would be good to start documenting these functions more thoroughly for development purposes.